libssu/ssurepomanager.cpp

```diff
@@ -470,16 +475,14 @@ QString SsuRepoManager::url(const QString &repoName, bool rndRepo,
     }
 
     // search for URLs for repositories. Lookup order is:
-    // 1. overrides in ssu.ini
```

Comment:
Would this not break use cases where the user want's to override a system repo? E.g. for development.

Reply:
That's the point of this. People were overriding system repos, forgetting, then their upgrades were failing. There's no need to override a system repo for development. You can just add a user one.

Reply:
Yes correct like @abranson said _"You can just add a user one."_

Reply:
I get it but sometimes having the original repo is not what is intended e.g. because the repo to be overriden pulls in packages you don't want, if we don't want overrides we can just remove them entirely.

Reply:
The original bug is a better place to discuss whether or not it should be fixed. It'll reach more people.

Reply:
I ended up only blocking removal, as the global repos defined in the ssu config should always be present. I originally blocked disabling too, but as there's a section in ssu lr for disabled global repos then I thought it best to still allow this.